CHANGELOG.md

```diff
@@ -963,7 +963,7 @@
 - [#30] - Create BambooTracker.1 (thanks [@alexmyczko], [@OPNA2608])
 - [#37], [#38] - Support WOPN bank and instrument format (thanks [@jpcima])
 - Add S98 export
-- [#13] - Add selection to move the cursor to right in the effect columns (thanks [@jimbo1qaz])
+- [#13] - Add selection to Move the cursor right in the effect columns (thanks [@jimbo1qaz])
```

Comment:
Additional "to" is fine, but why the capitalised "Move"? In general, maybe we shouldn't edit changelog lines from the distant past though.

Reply:
This change is a mistake. Must have included it by mistake in a global find/replace. Thanks for the catch.